jest.config.js

```diff
@@ -1,15 +1,16 @@
 module.exports = {
+  roots: ['<rootDir>/src'],
   globals: {
-    PROJECT_VERSION: '42.0.0',
+    PROJECT_VERSION: JSON.stringify(require('./package.json').version),
     KEYCLOAK_HOST: 'localhost',
     KEYCLOAK_REALM: 'SHOGun',
     KEYCLOAK_CLIENT_ID: 'shogun-client'
   },
   transform: {
-    '^.+\\.jsx?$': '<rootDir>/node_modules/babel-jest',
-    '^.+\\.tsx?$': '<rootDir>/node_modules/babel-jest'
+    '^.+\\.jsx?$': 'babel-jest',
```

Comment:
Out of curiosity, why does this need to be adjusted?

Reply:
It is not strictly neccessairy, just more simple/clean.